sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -1,17 +1,19 @@
+# Standard
```

Comment:
Do we need these comments in all the files? I think it is pretty standard to have this separation, so I'm not sure we need the comments in every file we are importing from.